internal/sidekick/internal/rust_release/find_cargo_manifests_test.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package rustrelease
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestFindManifest(t *testing.T) {
+	setupForVersionBump(t, "find-manifest-0.0.1")
```

Comment:
consider a test for the case where we have multiple `Cargo.toml` on the path, and we stop at the outermost one.

I am specifically thinking of changes to `echo-server` which should not necessarily lead to a `gax-internal` version bump.

https://github.com/googleapis/google-cloud-rust/tree/main/src/gax-internal/echo-server

Reply:
unnecessary if `release-plz` knows better than to bump the versions of `gax-internal` in that case